data/local-sqldelight/src/commonMain/sqldelight/com/adammcneilly/pocketleague/sqldelight/LocalEventStage.sq

```diff
@@ -15,6 +15,23 @@ CREATE TABLE localEventStage(
     FOREIGN KEY (eventId) REFERENCES localEvent(id) ON DELETE CASCADE
 );
 
+insertOrIgnoreEventStage:
```

Comment:
We're gonna want to do more of this, especially when we get less info. Usually happens when we query related items, like matches. So basically we should insert or ignore except for the primary item we're looking at.